### DISCUSSION TITLE
No occupancy message from Tradfri motion sensor?

### DISCUSSION THREAD

**Discussion by flatsiedatsie** [Koenkk/zigbee2mqtt#6024] [Devices]
I have an IKEA motion sensor that I'm trying to integrate into the WebThings Gateway smart home controller. Using the excellent Expose API I can see that it has an occupancy property.

After succesfully pairing the device, I see messages from this device telling me the battery level and connection strength. So far so good.

However, I never see incoming messages with this occupancy property?

I've tried this with two of these sensors. I've also noticed this with the IKEA on/off buttons, where I see general status messages about battery level, but don't receive any incoming messages when I press them.

Is this normal?

These devices were originally designed to pair with lightbulbs directly. Could that be the reason that I don't see state messages appear through Zigbee2MQTT? Am I perhaps supposed to add them to groups?

This is using a CC2531 with default Sonoff firmware that I bought online. I am able to toggle IKEA lights just fine.

**Comment by Kat-CeDe** [comment]
Hi,
they are sending occupancy. I have three. The blind time is 3 minutes. Did you check the log? Interview completed?

Ralf

**Comment by flatsiedatsie** [comment]
Thanks for responding. It turned out my Sonoff CC2531 USB stick's firmware wasn't passing these particular values through. I tested with a Conbee 2 and suddenly all values did indeed show up.
